src/patternfly/layouts/Bullseye/docs/code.md

```diff
@@ -1,10 +1,10 @@
 ## Overview
 
-A bullseye layout is when there is a single child element, and that child is centered both vertically and horizontally in the parent.
+Use a bullseye layout when there is a single child element, and that child is centered both vertically and horizontally in the parent.
 
 ## Usage
 
-| Class | Usage |
-| -- | -- |
-| `.pf-l-bullseye` |  **Outcome:** Initializes the bullseye layout | **Required:** Yes | **Remarks:** A bullseye can only have one child
-| `.pf-l-bullseye__item` |  **Outcome:** Explicitly sets the child for the bullseye  | **Required:** No | **Remarks:** This class isn't necessary, but it is included to keep inline with BEM convention, and to provide an entity that will later be used for applying modifiers.
+| Class | Applied | Outcomete |
+| -- | -- | -- |
+| `.pf-l-bullseye` | `<div>` | Initializes the bullseye layout. A bullseye can only have one child. |
```

Comment:
Throughout the docs/code.md files, there is a mix of "Initializes" and "Initiates". I'm not even sure if one is more appropriate than the other in this context, but should we be consistent? (I'm not in the dropdown docs, I used "defines")